common/test/acceptance/tests/lms/test_learner_profile.py

```diff
@@ -713,29 +733,52 @@ def test_different_user_public_profile(self):
         Then I shouldn't see the profile visibility selector dropdown.
         Also `location`, `language` and `about me` fields are not editable.
         """
-        different_username, different_user_id = self._initialize_different_user(privacy=self.PRIVACY_PUBLIC)
+        different_username, different_user_id = self.initialize_different_user(privacy=self.PRIVACY_PUBLIC)
         username, __ = self.log_in_as_unique_user()
         profile_page = self.visit_profile_page(different_username)
         profile_page.wait_for_public_fields()
         self.verify_profile_page_is_public(profile_page, is_editable=False)
         self.verify_profile_page_view_event(username, different_user_id, visibility=self.PRIVACY_PUBLIC)
 
-    def _initialize_different_user(self, privacy=None, birth_year=None):
+
+@attr('a11y')
+class LearnerProfileAxsTest(LearnerProfileTestMixin, WebAppTest):
```

Comment:
`axs` is the name used in the google accessibility ruleset.  We were using that originally, but have since switched to using the non-ruleset specific naming `a11y`.